### PR TITLE
fixing some typos in the README.md file

### PR DESCRIPTION
**Pull request by firasmlt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,7 +240,7 @@ The contents of the User-Agent header of the request.
 
 ### morgan.compile(format)
 
-Compile a format string into a `format` function for use by `morgan`. A format string
+Compiles a format string into a `format` function for use by `morgan`. A format string
```
Comment (dougwilson) [inline README.md:243]: This isn't a typo. If you still believe so, please elaborate.

File: README.md [hunk 2]
```diff
@@ -218,7 +218,7 @@ include on the number, defaulting to `3`, which provides microsecond precision.
 
 The status code of the response.
 
-If the request/response cycle completes before a response was sent to the
+If the request/response cycle ends before a response was sent to the
```
Comment (dougwilson) [inline README.md:221]: This isn't a typo. If you still believe so, please elaborate.

**Review by dougwilson — CHANGES_REQUESTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (firasmlt) [reply README.md:243]: I will undo it

File: README.md (hunk 2 above)
Comment (firasmlt) [reply README.md:221]: I will undo it

**Comment by firasmlt** [comment]
Done

**Comment by firasmlt** [comment]
@dougwilson can you review it again?

**Pull request closed without merging**